### PR TITLE
Is it "newskie" or "newsky" 🤔 

### PR DESCRIPTION
**Pull request by estrattonbailey**
CleanShot 2024-06-18 at 11 32 36@2x
CleanShot 2024-06-18 at 11 32 59@2x

### CODE REVIEW

**Comment by surfdude29** [comment]
**newsky** singular, **newskies** plural? 🤔

File: assets/icons/newskie.svg
Comment (haileyok) [inline assets/icons/newskie.svg:1]: sick, tysm

**Review by haileyok — APPROVED** [review]

File: src/components/NewskieDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import {Text} from '#/components/Typography'
+
+export function NewskieDialog({
+  profile,
+}: {
+  profile: AppBskyActorDefs.ProfileViewDetailed
+}) {
+  const {_} = useLingui()
+  const control = useDialogControl()
+  const profileName = profile.displayName || `@${profile.handle}`
```
Comment (estrattonbailey) [inline src/components/NewskieDialog.tsx:24]: Ah shoot I guess I should moderate this

**Comment by haileyok** [comment]
> **newsky** singular, **newskies** plural? 🤔

still figuring that out 😅

**Comment by estrattonbailey** [comment]
Can edit naming later if we want lol

**Pull request merged**